docker-compose.yml

```diff
@@ -298,14 +290,6 @@ services:
 
       # Utilize custom metadata
       - ./development/sp2-local/metadata/saml20-idp-remote.php:/data/vendor/simplesamlphp/simplesamlphp/metadata/saml20-idp-remote.php
-
-      # Local modules
-      - ./modules/mfa:/data/vendor/simplesamlphp/simplesamlphp/modules/mfa
-      - ./modules/expirychecker:/data/vendor/simplesamlphp/simplesamlphp/modules/expirychecker
-      - ./modules/profilereview:/data/vendor/simplesamlphp/simplesamlphp/modules/profilereview
-      - ./modules/silauth:/data/vendor/simplesamlphp/simplesamlphp/modules/silauth
-      - ./modules/sildisco:/data/vendor/simplesamlphp/simplesamlphp/modules/sildisco
-      - ./modules/material:/data/vendor/simplesamlphp/simplesamlphp/modules/material
```

Comment:
Are you sure about these local modules being removed?

Reply:
Yes, I shouldn't have added them to begin with. The SP containers are pretty bare-bones. They are only there to receive login requests and redirect to the Hub and/or IdP containers. Our custom modules are all for IdP and Hub use.